### PR TITLE
cleaned up html file and added a css file 

### PR DESCRIPTION
**Pull request by wijiler**
css files are cool

### CODE REVIEW

**Comment by wijiler** [comment]
yarn.lock file changed a bit

File: README.md [hunk 1]
```diff
@@ -19,10 +19,10 @@ yarn install
 ```
 
 You will also need to install the game depenencies
+<!--- Dumbest change I could ever make but its a change --->
 
 ```
-cd player
-yarn install
+cd player && yarn install
```
Comment (cmgriffing) [inline README.md]: This is not compatible with windows.

**Comment by cmgriffing** [comment]
> yarn.lock file changed a bit

Hmm, I would rather not add unnecessary yarn changes. is that because we did yarn install rather than just yarn?

**Comment by wijiler** [comment]
> > yarn.lock file changed a bit
> 
> Hmm, I would rather not add unnecessary yarn changes. is that because we did yarn install rather than just yarn?

it happened when i installed dependencies

File: README.md (hunk 1 above)
Comment (wijiler) [reply README.md]: Ill get on that

**Comment by wijiler** [comment]
sorry I havent committed the changes I got sick :(

**Comment by wijiler** [comment]
ill be comittting them soon :D!

**Pull request closed without merging**